sd/test/equip_test.sh

```diff
@@ -63,7 +62,12 @@ get_config() {
     grep $1 /home/hd1/test/yi-hack.cfg  | cut -d"=" -f2
 }
 
-
+boot_voice() {
+    voice_file=$1
+    if [ "$(get_config BOOT_VOICE)" = "yes" ]; then
```

Comment:
In this scenario, `=` and `==` are equivalent 😄 . See http://tldp.org/LDP/abs/html/comparison-ops.html#SCOMPARISON1.